config/routes.rb

```diff
@@ -15,6 +15,7 @@
         format: false
   end
 
+  get "/:id/clear_session", to: "session_answers#destroy", as: :delete_session_flow
```

Comment:
Nit: Think it'd be clearer to use a single consistent verb here, instead of "clear",  "destroy" and "delete"

Reply:
Hi @theseanything - thanks.  Route renamed, tests updated.  Link updated - in later commit.